src/modules/job-archive/job-archive.c

```diff
@@ -499,12 +499,26 @@ void job_archive_cb (flux_reactor_t *r,
                      void *arg)
 {
     struct job_archive_ctx *ctx = arg;
-    char *attrs = "[\"userid\", \"ranks\", \"t_submit\", " \
-                   "\"t_run\", \"t_cleanup\", \"t_inactive\"]";
     flux_future_t *f;
 
-    if (!(f = flux_job_list_inactive (ctx->h, 0, ctx->since, attrs))) {
-        flux_log_error (ctx->h, "%s: flux_job_list_inactive", __FUNCTION__);
+    if (!(f = flux_rpc_pack (ctx->h,
```

Comment:
Similar suggestion for the other commit message - instead of "remove flux_job_list() warnings", just "do not use flux_job_list()"? Eh, take or leave these suggestions.